docs/src/main/asciidoc/metrics-guide.adoc

```diff
@@ -0,0 +1,180 @@
+include::./attributes.adoc[]
+= {project-name} - MicroProfile Metrics
+
+This guide demonstrates how your {project-name} application can utilize the MicroProfile
+Metrics specification through the SmallRye Metrics extension.
+
+MicroProfile Metrics allows applications to gather various metrics and statistics that provide
+an insight into what is happening inside the application.
+
+The metrics can be read remotely using JSON format or the OpenMetrics format, so that
+they can be processed by additional tools such as Prometheus, and stored for analysis
+and visualisation.
+
+== Prerequisites
+
+To complete this guide, you need:
+
+* less than 15 minutes
+* an IDE
+* JDK 1.8+ installed with `JAVA_HOME` configured appropriately
+* Apache Maven 3.5.3+
+
+== Architecture
+In this example, we build a very simple microservice which offers one REST endpoint and that endpoint serves
+for determining whether a number is prime. The implementation class is annotated with some metric annotations
+so that while responding to user's requests, some metrics will be gathered. The meaning of each metric will be explained later.
+
+== Solution
+
+We recommend that you follow the instructions in the next sections and create the application step by step.
+However, you can go right to the completed example.
+
+Clone the Git repository: `git clone {quickstarts-clone-url}`, or download an {quickstarts-archive-url}[archive].
+
+The solution is located in the `microprofile-metrics` {quickstarts-tree-url}/microprofile-metrics[directory].
+
+== Creating the Maven Project
+
+First, we need a new project. Create a new project with the following command:
+
+[source, subs=attributes+]
+----
+mvn io.quarkus:quarkus-maven-plugin:0.11.0:create \
```

Comment:
It preferable to use the maven wrapper so I propose to change the `mvn` command with `./mvnw`

Reply:
Maven plugin 0.11.0 doesn't create a Maven wrapper though, I see that this was added recently so it will be created with 0.12.0. So what now? Should we keep this as is and then change the `mvn` command to `./mvnw` when we're updating the plugin version to 0.12.0?

Reply:
the maven-tooling doc in master have already been changed to use `./mvnw` so i think its best if we do that here as well.

Reply:
Ok I changed `mvn` to `./mvnw` in the `clean compile quarkus:dev` command. Not for the `quarkus-maven-plugin:0.11.0:create` command because there is no wrapper available at that time yet.